programs/cli_wallet/main.cpp

```diff
@@ -68,45 +68,59 @@ using namespace graphene::wallet;
 using namespace std;
 namespace bpo = boost::program_options;
 
-/***
- * @brief sets up basic logging
- * 
- * Sends most logging messages (up to level "info") to the console and RPC messages 
- * (including level "debug")to a file
- */
-void setup_logging()
+fc::log_level string_to_level(string level)
+{
+   fc::log_level result;
+   if(level == "info")
+      result = fc::log_level::info;
+   else if(level == "debug")
+      result = fc::log_level::debug;
+   else if(level == "warn")
+      result = fc::log_level::warn;
+   else if(level == "error")
+      result = fc::log_level::error;
+   else if(level == "all")
+      result = fc::log_level::all;
+
```

Comment:
Please throw if none of the above.

Reply:
done at https://github.com/bitshares/bitshares-core/pull/1814/commits/c793ef24433cbc2fe76b1927cc7846892ccbba0a